control/excalibur/hl_detector.py

```diff
@@ -1322,6 +1346,10 @@ def get(self, path):
                 response = {'value': 1}
             elif path == 'command/configure_mask':
                 response = {'value': 1}
+            elif path == 'command/pause_polling':
+                response = {'value': 1}
+            elif path == 'command/continue_polling':
+                response = {'value': 1}
```

Comment:
Unrelated to this PR specifically, but why aren't the commands mapped through the parameter tree rather than requiring a `if/elif/else` dispatch in both the `get` and `set` methods?

Reply:
When the migration to parameter tree took place, these might have been missed as they are handled in a different part of the original system.  I think they should probably be refactored into the parameter tree, I'll take a look and raise a new ticket assuming there's no reason not to do it.